string.c

```diff
@@ -8252,10 +8252,10 @@ rb_str_delete(int argc, VALUE *argv, VALUE str)
 
 /*
  *  call-seq:
- *     str.squeeze!([other_str]*)   -> str or nil
+ *    squeeze!(*selectors) -> new_string or nil
```

Comment:
I don't think the return value is a new string. It's the `self` string right?

Reply:
Fixed.